stdlib/src/builtin/simd.mojo

```diff
@@ -2538,11 +2538,11 @@ fn _min_finite[type: DType]() -> Scalar[type]:
         return -128
     elif type == DType.int16:
         return -32768
-    elif type == DType.int32 or (type == DType.index and _is_32_bit_system()):
+    elif type == DType.int32 or type.is_index32():
         return -2147483648
-    elif type == DType.int64 or (
-        type == DType.index and not _is_32_bit_system()
-    ):
+    elif type == DType.float32:
```

Comment:
**Suggestion** I think this rebase part is not correct.  We want to just fall through to the bottom part of `elif type.is_floating_point()` which we changed internally recently.

Reply:
oh i see